Week1/homework/theCatWalk/jscode.js

```diff
@@ -0,0 +1,44 @@
+
+let image;
```

Comment:
This variable should not be there. If I remember correctly you placed image here so you can view it from chrome devTools console. This should not be the case. If you want to view something in the console, you can just console.log if at the right place